src/dataset-builder/CohortEditor.test.ts

```diff
@@ -58,17 +62,21 @@ describe('CohortEditor', () => {
     asMockedFn(DataRepo).mockImplementation(() => mockDataRepoContract as DataRepoContract);
   };
 
+  const programDataRangeOption = (min = 55, max = 99): ProgramDataRangeOption => {
+    return {
+      id: 0,
+      kind: 'range',
+      name: 'range',
```

Comment:
why do we have both kind and name?

Reply:
oh to make it work with the domain option which will have a different name?

Reply:
This is just for tests. In the real data it will look more like `kind: 'range', name: 'Year of Birth'`